factcast-itests/factcast-itests-factus/src/test/java/org/factcast/itests/factus/client/SubscriptionReconnectionITest.java

```diff
@@ -51,9 +52,12 @@
 @SpringBootTest
 @ContextConfiguration(classes = TestFactusApplication.class)
 @TestPropertySource(
-    properties =
-        "factcast.grpc.client.resilience.attempts="
-            + SubscriptionReconnectionITest.NUMBER_OF_ATTEMPTS)
+    properties = {
+      "factcast.grpc.client.resilience.attempts="
+          + SubscriptionReconnectionITest.NUMBER_OF_ATTEMPTS,
+      // some tests here assume event handling happens sequential even on networking level
+      "factcast.grpc.client.catchup-batchsize=1"
```

Comment:
- flaky test `subscribeWithFailingReconnect` assumes it can have impact on event handling by introducing networking issues during the handling of a specific event
- we must make sure there is still something left to do that the introduced networking issue can actually affect